test/unit/cssTest.js

```diff
@@ -0,0 +1,356 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const path = require('path')
+const generateTestApp = require('../util/generateTestApp')
+const cleanupTestApp = require('../util/cleanupTestApp')
+const fork = require('child_process').fork
+const fs = require('fs')
+const fse = require('fs-extra')
+const klawsync = require('klaw-sync')
+
+// appDir
+const appDir = path.join(__dirname, '../', 'app', 'cssTest')
+
+// basic CSS files
+let cssDataArray = [
+  `body {
+  height: 100%;
+}
+h1 {
+  font-size: 10px;
+}
+`,
+  `#selector {
+  height: 100%;
+}
+h1 {
+  font-size: 10px;
+}
+`,
+  `.body {
+  height: 100%;
+}
+h1 {
+  font-size: 10px;
+}
+`
+]
+
+// version package json file
+let packageJSON = {
+  version: '0.3.1',
+  rooseveltConfig: {}
+}
+
+// path to CSS static files
+let pathOfCSSStaticFilesArray = [
+  path.join(appDir, 'statics', 'css', 'a.css'),
+  path.join(appDir, 'statics', 'css', 'b.css'),
+  path.join(appDir, 'statics', 'css', 'c.css')
+]
+
+// path to CSS compiled files
```

Comment:
rephrase this comment to: `array of paths to where compiled css files should be written`